### ISSUE TITLE
JEI recipe wrong for Abyssal Wasteland Staff of Rending

### ISSUE THREAD

**Issue by mrtopf** [Shinoow/AbyssalCraft#150]
I am trying to upgrade the initial staff of rending via the ritual in the abyssal wasteland.

In JEI it says it needs the necronomicon but it only works with the abyssal version.

**Comment by Shinoow** [comment]
Technically it does work with the regular, _except_ altar creation in the Abyssal Wasteland can't be done with the regular Necronomicon. The initial idea there was more or less to say "hey, you can upgrade the staff for more efficiency", only problem being that said upgrade required the next Necronomicon, so it pretty much backfired on itself.

**Comment by mrtopf** [comment]
Yeah, I found out about altar creation (the book only says to use "a necronomicon" not which one). Also it says "The essence is obtained through the use of a Staff of Rending". It doesn't really say though what the use is. I figured it out in the end but more information might be nice to have. I know though that this will make the book bigger and thus for some ppl harder to read (or it needs a more complex book implementation with links and maybe search). And right now I found it in JEI ;-)

As for the creation of the Abyssal Wasteland Staff of Rending I am pretty sure it only worked with the abyssal necronomicon and not the overworld one. I spent a bit of time checking what is wrong and in the end used the abyssal one.

For the Dreadlands Staff it says to use the Abyssal necronomicon, so apparently it's not always the same (according to JEI).

But I will try to create a new abyssal one and a dreadlands one later and see which version I need. 

I also don't think the staff backfired completely, you are just always a step behind as I now have an abyssal one in the dreadlands and that's of course better than having the initial one. Just the last version might be of no use then (unless you need essence for more than the book and bags someday).

**Comment by mrtopf** [comment]
So I tested making a dreadlands staff. I was in the dreadlands, put everything on the ritual but couldn't start it with the abyssal wasteland necronomicon. I could only start it with the dreadlands one. JEI nevertheless shows the abyssal one as the one to use. So from what I see the JEI recipes all need to be upgraded by one book (which IMHO also makes sense).

**Comment by Shinoow** [comment]
Like I said in my comment above, the idea behind those rituals backfired, since I have internal things keeping you from performing a ritual in a dimension that you can't make a Ritual Altar inside, which renders those rituals redundant, since they've been designed for the next dimension's book, while not registered for it.

**Comment by mrtopf** [comment]
I am not sure I understand it completely as I can perform the rituals but only after I made the new book for which the new staff would have been helpful. I think it also states the correct dimension in JEI. 

But as long as you know what the problem is that's the important part :-) So feel free to close it.

**Comment by Shinoow** [comment]
Fixed in 1.9.3.4
